### PR TITLE
added input focus helper

### PR DESCRIPTION
**Pull request by weslers**
Can you guys test this locally to make sure the autofocus is working. The autofocus HTML attr wont work I think it is due to the way materielize use JS with the float label appearance.

### CODE REVIEW

File: client/templates/initiative/create/initiativeCreate.js [hunk 1]
```diff
@@ -53,4 +53,11 @@ Template.initiativeCreate.events({
       }
     });
   }
+});
+
+Template.initiativeCreate.onRendered(function(){
```
Comment (jeffshaver) [inline client/templates/initiative/create/initiativeCreate.js]: will fail eslint. needs a space before the `{`

File: client/templates/initiative/create/initiativeCreate.js [hunk 2]
```diff
@@ -53,4 +53,11 @@ Template.initiativeCreate.events({
...
+    var input = this.find('#name');
+    if(input){
```
Comment (jeffshaver) [inline client/templates/initiative/create/initiativeCreate.js]: will fail eslint. needs a space after `if` and a space before the `{`

File: client/templates/app/signup/signup.js [hunk 3]
```diff
@@ -36,4 +36,11 @@ Template.publicSignup.events({
       Router.go('initiatives');
     });
   }
+});
+
+Template.publicSignup.onRendered(function(){
+    var input = this.find('.email-address-input');
+    if(input){
```
Comment (jeffshaver) [inline client/templates/app/signup/signup.js]: will fail eslint. needs a space after `if` and a space before the `{`

File: client/templates/app/signup/signup.js [hunk 4]
```diff
@@ -36,4 +36,11 @@ Template.publicSignup.events({
       Router.go('initiatives');
     });
   }
+});
+
+Template.publicSignup.onRendered(function(){
```
Comment (jeffshaver) [inline client/templates/app/signup/signup.js]: will fail eslint. needs a space before the `{`

File: client/templates/app/search/search.js [hunk 5]
```diff
@@ -16,4 +16,11 @@ Template.search.helpers({
   isSearching: function isSearching() {
     return Session.get('searchTerm') !== '';
   }
+});
+
+Template.search.onRendered(function(){
+    var input = this.find('#search_initiatives');
+    if(input){
```
Comment (jeffshaver) [inline client/templates/app/search/search.js]: will fail eslint. needs a space after `if` and a space before the `{`

File: client/templates/app/search/search.js [hunk 6]
```diff
@@ -16,4 +16,11 @@ Template.search.helpers({
   isSearching: function isSearching() {
     return Session.get('searchTerm') !== '';
   }
+});
+
+Template.search.onRendered(function(){
```
Comment (jeffshaver) [inline client/templates/app/search/search.js]: will fail eslint. needs a space before the `{`

File: client/templates/app/login/login.js [hunk 7]
```diff
@@ -54,4 +54,11 @@ Template.publicLogin.events({
       }
     });
   }
+});
+
+Template.publicLogin.onRendered(function(){
+    var input = this.find('.email-address-input');
+    if(input){
```
Comment (jeffshaver) [inline client/templates/app/login/login.js]: will fail eslint. needs a space after `if` and a space before the `{`

File: client/templates/app/login/login.js [hunk 8]
```diff
@@ -54,4 +54,11 @@ Template.publicLogin.events({
       }
     });
   }
+});
+
+Template.publicLogin.onRendered(function(){
```
Comment (jeffshaver) [inline client/templates/app/login/login.js]: will fail eslint. needs a space before the `{`

**Comment by jeffshaver** [comment]
Commented on the places I believe this will fail eslint as a follow up to @mariocatch's comment

**Comment by weslers** [comment]
I have made those updates now

**Comment by jeffshaver** [comment]
For consistency with other code, please invert the if's.

```
if (_.isEmpty(input)) {
  return;
}
$(input).focus();
```

**Comment by weslers** [comment]
so 

``` javascript
Template.publicLogin.onRendered(function() {
    var input = this.find('.email-address-input');
    if (input) {
        input.focus();
    }
});
```

should be 

``` javascript
Template.publicLogin.onRendered(function() {
    var input = this.find('.email-address-input');
    if (_.isEmpty(input)) {
      return;
    }
    $(input).focus();
});
```

**Comment by jeffshaver** [comment]
Just following up is that the agreed upon change here is:

```
Template.publicSignup.onRendered(function() {
  var input = this.find('.email-address-input');
  if (!input) {
    return;
  }
  input.focus();
});
```

**Comment by jeffshaver** [comment]
There is a rule that requires a new line between var declarations and new code. There is also a rule that requires functions to be named. So this:

```
Template.publicLogin.onRendered(function() {
  var input = this.find('.email-address-input');
  if (!input) {
    return;
  }
  input.focus();
});
```

 would change to:

```
Template.publicLogin.onRendered(function publicLoginOnRendered() {
  var input = this.find('.email-address-input');

  if (!input) {
    return;
  }
  input.focus();
});
```

**Pull request merged**